docs/upgrade.html

```diff
@@ -85,6 +85,7 @@
 <h5><a id="upgrade_10_notable" href="#upgrade_10_notable">Notable changes in 0.10.0.0</a></h5>
 
 <ul>
+    <li> Starting from Kafka 0.10.0.0, a new client library named <b>KafkaStreams</b> is added for stream processing on data stored in Kafka brokers. This new client library only works with 0.10.x and upward versioned brokers due to message format changes mentioned above. For more information please read <a href="#streams_overview">this section</a>.</li>
```

Comment:
"is added" -> "is available"

"for stream processing on data stored in Kafka brokers" -> "for stream processing on data stored in Kafka topics" (brokers is correct, too, but I think "topics" is clearer).